CHANGELOG.develop

```diff
@@ -803,6 +803,7 @@ Other:
     (thanks to rayw000)
   - Added =string-edit= package to =spacemacs-editing= layer
     (thanks to Ivan Yonchovski)
+  - Added =org-appear= package to =spacemacs-org= layer (thanks to winsphinX)
```

Comment:
Should this be under the Org layer instead of in "Distribution changes"?

Reply:
Yes indeed, please move this comment to the org layer section.